bcgov/HOUS-permit-portal
# allow large file multi upload, test with > 10MB files

## Description

Allow large file uploads

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-674

## Steps to QA

In form:
Try to upload a file < 10 MB, there should be no error, and in the network tab you should see one PUT request to the bucket
Try to upload a file > 10 MB, < 100 MB, there should be no error, you should see multiple PU
t requests in the network tab
Try downloading the files that were uploaded.  They should match the size that was uploaded.
Try to upload a file > 100 MB, it should give you an error saying its not possible.

Try uploading a small h2k file, it should work.  Chacnes are h2k will never hit the large file.
